app/propel/migrations/PropelMigration_1503510123.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * Data object containing the SQL and PHP code to migrate the database
+ * up to version 1503510123.
+ * Generated on 2017-08-23 20:42:03 by kasutaja
+ */
+class PropelMigration_1503510123
+{
+
+    public function preUp($manager)
+    {
+        // add the pre-migration code here
+    }
+
+    public function postUp($manager)
+    {
+        // add the post-migration code here
+    }
+
+    public function preDown($manager)
+    {
+        // add the pre-migration code here
+    }
+
+    public function postDown($manager)
+    {
+        // add the post-migration code here
+    }
+
+    /**
+     * Get the SQL statements for the Up migration
+     *
+     * @return array list of the SQL strings to execute for the Up migration
+     *               the keys being the datasources
+     */
+    public function getUpSQL()
+    {
+        return array (
+  'default' => '
+# This is a fix for InnoDB in MySQL >= 4.1.x
+# It "suspends judgement" for fkey relationships until are tables are set.
+SET FOREIGN_KEY_CHECKS = 0;
+
+ALTER TABLE `ollekassa_report` DROP FOREIGN KEY `member_fk`;
```

Comment:
Huvitav, miks sul selline muudatus siia tekkis? Minul ega live baasis ei ole sellist Foreign Key'd. Ei saagi olla, sest liikmete tabel on MyISAM.

Reply:
Võta see migrast maha, kui sul omal oli kuidagi see foreign key, siis lase ainult endal käima.
Migratsioonide käivitamine andiski mulle vea:
Failed to execute SQL "ALTER TABLE `ollekassa_report` DROP FOREIGN KEY `member_fk`"